### PR TITLE
created PatientBannerModel, PatientBannerController, endpoint and also tested in postman 

### PR DESCRIPTION
**Pull request by lorenaanghel**
Everything works properly.

### CODE REVIEW

**Review by andreeacojocaru731 — CHANGES_REQUESTED** [review]
Please remove the bin folder, it should not be committed.

File: patient-view-be/src/main/java/spvproject/patientviewbe/model/PatientBannerModel.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package spvproject.patientviewbe.model;
+
+
+public class PatientBannerModel {
```
Comment (sweetphilosophy) [inline patient-view-be/src/main/java/spvproject/patientviewbe/model/PatientBannerModel.java]: This can be simply called `PatientBanner`

**Review by sweetphilosophy — CHANGES_REQUESTED** [review]
Let me know your thoughts!
Also, please make sure to remove the `/bin/` folder as Andreea mentioned. I think you didn't merged master into your branch before working? I could be wrong!
In any case, only the source files should be committed

File: patient-view-be/src/main/java/spvproject/patientviewbe/controller/PatientBannerController.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package spvproject.patientviewbe.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import spvproject.patientviewbe.model.PatientBanner;
+
+@RestController
+@RequestMapping("GET/patient-banner")
```
Comment (sweetphilosophy) [inline patient-view-be/src/main/java/spvproject/patientviewbe/controller/PatientBannerController.java]: You can remove the `GET` from here.

**Review by sweetphilosophy — APPROVED** [review]
Looks goo to me. Thanks for following up the comments

**Review by andreeacojocaru731 — APPROVED** [review]

**Pull request merged**